InsertionSort.java

```diff
@@ -0,0 +1,20 @@
+package sortBench.algorithm;
+
+public class InsertionSort implements ISortingAlgorithm{ 
+     
+    public static int[] doInsertionSort(int[] input){
```

Comment:
To be compliant with sort method define on ISortingAlgorithm you rename this method, remove the static part return void instead of int[]

`public void sort(int[] input) {`